notebooks/contributing_index.md

```diff
@@ -0,0 +1,40 @@
+Contributing to User Tutorials
+==============================
+
+SimPEG welcomes contributions by the community to improve available tutorial materials.
+However, tutorials must adhere to specific guidelines in order to ensure a high level of quality and consistency.
+For those interested in contributing to SimPEG User Tutorials, please follow the instructions provided.
+
+
+Before Creating a New Tutorial
+------------------------------
+
+```{admonition} WARNING!
+:class: danger
+Please ensure that the contents of your tutorial **DOES NOT** fall into any of the following categories:
+
+* the tutorial focusses on functionality that is not part of SimPEG
+* the amount of SimPEG functionality NOT already covered in another tutorial is insufficient
+* the tutorial focusses on a data-specific result and is not generalizable
+* the tutorial was created for the purpose of self promotion
```

Comment:
I struggle a little bit with this point. If any contributor wants to add a tutorial and the tutorial is great, the reasons why they want to push it should not matter too much in order to decide whether we want to merge it or not. Also, how can we prove that a given contributor opened the PR with the purpose of self promotion? Answers to that question can only be subjective.

I'd be inclined to remove this point, or keep it but provide a clear definition of "self promotion" that could be objectively determined.

What do you think? What type of contributions is this rule trying to avoid?